spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/BaseCatalog.java

```diff
@@ -47,4 +56,38 @@ public Procedure loadProcedure(Identifier ident) throws NoSuchProcedureException
 
     throw new NoSuchProcedureException(ident);
   }
+
+  @Override
+  public Identifier[] listFunctions(String[] namespace) throws NoSuchNamespaceException {
+    if (namespace.length == 0
+        || (namespace.length == 1 && namespace[0].equalsIgnoreCase("system"))) {
```

Comment:
nit: What about defining a helper function `isSystemNamespace`? There are 3 places in this class where we can call it.

```
private boolean isSystemNamespace(String[] namespace) {
  return namespace.length == 1 && namespace[0].equalsIgnoreCase("system");
}
```

Reply:
It is also used in the method for loading procedures.

Reply:
Cool. Updated all 3. This helps with the funky formatting too. Thank you!